subxt/examples/blocks_subscribing.rs

```diff
@@ -34,6 +34,17 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
             let decoded_ext = ext.as_root_extrinsic::<polkadot::Call>();
 
             println!("    Extrinsic #{idx}:");
+            if let Some(signed_extensions) = ext.signed_extensions() {
+                println!("      Signed Extensions:");
+                for signed_extension in signed_extensions.iter() {
+                    let signed_extension = signed_extension?;
+                    let name = signed_extension.name();
+                    let value = signed_extension.value()?.to_string();
+                    println!("        {name}: {value}");
+                }
+            } else {
+                println!("      No Signed Extensions");
+            }
```

Comment:
Just because this is an example, I wonder whether we could make this a bit less ugly, ie removing the `else` branch and moving it after the other `println`'s or something?